pydatalab/pydatalab/models/people.py

```diff
@@ -91,6 +92,21 @@ def add_missing_type(cls, v):
     def set_default_type(cls, _):
         return "people"
 
+    @validator("display_name")
+    def validate_display_name_length(cls, v):
+        """Validate that the display name."""
+        if len(v) > 150:
+            raise ValueError("Display name must be at most 150 characters long.")
+        return v
+
+    @validator("contact_email")
+    def validate_contact_email_format(cls, v):
+        """Validate that the contact email has a valid email format."""
+        email_regex = re.compile(r"[^@]+@[^@]+\.[^@]+")
+        if v is not None:
+            not email_regex.match(EmailStr(v))
+        return v
```

Comment:
I think this one is actually unnecessary because the `contact_email` field in the model already has the pydantic type `EmailStr` (which defines a regexp too). In this case, we could use this type directly in the user route to check that we are updating the db with an email.